Tests/TuistLoaderTests/Models+ManifestMappers/TargetScript+ManifestMapperTests.swift

```diff
@@ -33,24 +33,97 @@ final class TargetScriptManifestMapperTests: TuistUnitTestCase {
         // Given
         let temporaryPath = try temporaryPath()
         let generatorPaths = GeneratorPaths(manifestDirectory: temporaryPath)
+        try createFiles([
+            "foo/bar/a.swift",
+            "foo/bar/b.swift",
+            "foo/bar/aTests.swift",
+            "foo/bar/bTests.swift",
+            "foo/bar/kTests.kt",
+            "foo/bar/c/c.swift",
+            "foo/bar/c/cTests.swift",
+        ])
+
+        let manifest = ProjectDescription.TargetScript.test(
+            name: "MyScript",
+            tool: "my_tool",
+            order: .pre,
+            arguments: ["arg1", "arg2"],
+            inputPaths: ["foo/bar/**/*.swift"],
+            inputFileListPaths: ["$(SRCROOT)/foo/bar/**/*.swift"],
+            outputPaths: ["$(SRCROOT)/foo/bar/**/*.swift"],
+            outputFileListPaths: ["$(SRCROOT)/foo/bar/**/*.swift"]
+        )
+        // When
+        let model = try TuistGraph.TargetScript.from(manifest: manifest, generatorPaths: generatorPaths)
+
+        // Then
+        let relativeSources = model.inputPaths.map { $0.relative(to: temporaryPath).pathString }
+
+        XCTAssertEqual(Set(relativeSources), Set([
+            "foo/bar/a.swift",
+            "foo/bar/b.swift",
+            "foo/bar/aTests.swift",
+            "foo/bar/bTests.swift",
+            "foo/bar/c/c.swift",
+            "foo/bar/c/cTests.swift",
+        ]))
```

Comment:
What if we extract the list of paths into a variable to reuse it in `createFiles` and here?

Reply:
I can do that but each list is unique. For example, the input list contains `.kt` file which I did not want to remove to be honest. It is another check when we do `*.swift`. So, it seemed a bit redundant to have another variable in this case.